spring-aot-test/src/main/java/org/springframework/aot/test/context/bootstrap/generator/TestContextAotProcessor.java

```diff
@@ -110,7 +116,7 @@ protected ClassName generateTestContext(BootstrapWriterContext writerContext, Su
 		return mainBootstrapClass.getClassName();
 	}
 
-	private MethodSpec.Builder contextLoadersMappingMethod(Map<ClassName, TestContextConfigurationDescriptor> entries) {
+	private MethodSpec.Builder getContextLoadersBuilder(Map<ClassName, TestContextConfigurationDescriptor> entries) {
```

Comment:
I am using such a pattern for all `MethodSpec.Builder` within the code base and not using the term "builder" in the name of the method was done on purpose as this is a private detail. The same goes for `get` that was removed. 

We can discuss any style change but this has to be done consistently and not as part of this PR.

Reply:
OK. I've reverted that and renamed the new method to `contextInitializersMappingMethod` for consistency.